src/components/ClientWrapper/ChallengeRequestClient.tsx

```diff
@@ -104,35 +101,59 @@ export default function ChallengeRequestClient() {
       mediaType: formattedMediaType,
       imageCount: images.length
     };
-    console.log(data);
 
     try {
-      if (!accessToken) {
-        alert('User is not authenticated.');
-        return;
-      }
-
-      const res = await fetchChallengeRequest(data, accessToken);
+      const res = await fetchChallengeRequest(data);
       const { challenge, uploadUrls } = res;
+      const uploadResults = await Promise.all(
+        images.map(async (image, index) => {
+          const uploadUrl = uploadUrls[index]?.uploadUrl;
+          if (!uploadUrl) {
+            return {
+              success: false,
+              error: 'Upload URL not provided',
+              index
+            };
+          }
 
-      await Promise.all(
-        images.map((image, index) => {
-          const uploadUrl = uploadUrls[index];
-          return fetch(uploadUrl, {
-            method: 'PUT',
-            body: image,
-            headers: {
-              'Content-Type': image.type
+          try {
+            const response = await fetch(uploadUrl, {
+              method: 'PUT',
+              body: image,
+              headers: {
+                'Content-Type': image.type || 'application/octet-stream'
+              }
+            });
+
+            if (!response.ok) {
+              return {
+                success: false,
+                error: `HTTP ${response.status}: ${response.statusText}`,
+                index
+              };
             }
-          });
+            return {
+              success: true,
+              index
+            };
+          } catch (error) {
+            return {
+              success: false,
+              error,
+              index
+            };
+          }
         })
       );
-
-      alert('Form submitted successfully!');
+      const failedUploads = uploadResults.filter(result => !result.success);
+      if (failedUploads.length > 0) {
```

Comment:
이 부분 저번에 멘토님께서 truthy, falsy 개념 사용하라고 하셨어서,
```suggestion
if (faildUploads?.length) {
  alert('Some images failed to upload. Please try again.');
  return;
}
```
이런 식으로 작성해주세요!

Reply:
git 이상해서 다음 branch에서 작업하겠습니다 !